components/AlbumEdit/ChangeCoverPhoto.tsx

```
@@ -0,0 +1,79 @@
+type Props = {
+	albumId: string;
+	photos: Photo[];
+};
+
+export default function ChangeCoverPhoto({ albumId, photos }: Props) {
+	const [selectedPhoto, setSelectedPhoto] = useState("");
+	const dispatch = useDispatch<AppDispatch>();
+	const { currentUser } = useContext(authContext);
+	const uid = currentUser?.uid;
+
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: uid

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)